src/main/java/de/qabel/desktop/ui/contact/ContactController.java

```diff
@@ -95,6 +88,8 @@ public void initialize(URL location, ResourceBundle resources) {
         details = (DetailsController) detailsView.getPresenter();
         detailsView.getViewAsync(contactroot.getChildren()::add);
 
+        contactRepository.attach(() -> Platform.runLater(() -> update()));
```

Comment:
`() -> update()`  does  the same as `this::update` only with one runnable wrapper less. And as update already explicitly calls Platform.runLater, you can simplify this to
```JAVA
contactRepository.attach(this::update);
```